staging/src/k8s.io/dynamic-resource-allocation/kubeletplugin/draplugin.go

```diff
@@ -430,6 +430,18 @@ func DRAService(enabled bool) Option {
 	}
 }
 
+// ErrorChannel allows the plugin to send errors to the caller. This is
+// useful to report errors that happen in goroutines. The caller is expected
+// to read from the channel and handle errors appropriately.
+// For example, this channel can be used to report errors returned by the
+// grpcserver.Serve. Plugins then can use this error to perform a graceful shutdown.
+func ErrorChannel(errorChannel chan error) Option {
```

Comment:
```suggestion
func ErrorChannel(errorChannel chan<- error) Option {
```

The helper is only meant to write to this channel, not read from it.